### PR TITLE
fix(cloud-dedicated): get-started write

### PR DESCRIPTION
**Pull request by jstirnaman**
- Remove org param from examples where it's no longer required.
- Fix examples
- Add `Bearer` scheme

See individual commits.

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/get-started/write.md [hunk 1]
```diff
@@ -449,11 +447,14 @@ Replace the following:
 - **`DATABASE_NAME`**: your InfluxDB Cloud Dedicated database
 - **`DATABASE_TOKEN`**: a [database token](/influxdb/cloud-dedicated/admin/tokens/) with sufficient permissions to the database
 
-Install all the necessary packages and run the program to write the line
+Install the v2 client library and then run the program to write the line
 protocol to your InfluxDB Cloud Dedicated cluster.
 
 ```sh
-go get ./...
+go get github.com/influxdata/influxdb-client-go/v2
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/get-started/write.md:454]:
```suggestion
go get ./...
```

My bad. Didn't set it up properly before testing.

File: content/influxdb/cloud-dedicated/get-started/write.md [hunk 2]
```diff
@@ -175,8 +174,10 @@ export INFLUX_HOST=https://cluster-id.influxdb.io
 export INFLUX_ORG=ignored
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/get-started/write.md:174]:
```suggestion
```

File: content/influxdb/cloud-dedicated/get-started/write.md [hunk 3]
```diff
@@ -157,27 +157,31 @@ request to the InfluxDB API `/api/v2/write` endpoint using the `POST` request me
...
 The following example uses cURL and the InfluxDB v2 API to write line protocol
 to InfluxDB Cloud dedicated:
 
 {{% influxdb/custom-timestamps %}}
 ```sh
 export INFLUX_HOST=https://cluster-id.influxdb.io
-export INFLUX_ORG=ignored
 export INFLUX_TOKEN=DATABASE_TOKEN
 
+DATABASE_NAME=get-started
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/get-started/write.md:180]: Any reason why you export environment variables for the others, but create a shell variable for the database?

**Review by sanderson — APPROVED** [review]
Looks good. Just one question.

File: content/influxdb/cloud-dedicated/get-started/write.md (hunk 3 above)
Comment (jstirnaman) [reply content/influxdb/cloud-dedicated/get-started/write.md:180]: I guess for this example, none of them need to be exported.

**Pull request merged**